helm/eirini/templates/bits.yaml

```diff
@@ -130,7 +130,10 @@ spec:
           mountPath: /assets/
       initContainers:
       - name: "download-eirini-rootfs"
-        image: flintstonecf/eirinifs-downloader:latest
```

Comment:
Please switch to flintstonecf docker image. Thanks.

Reply:
done